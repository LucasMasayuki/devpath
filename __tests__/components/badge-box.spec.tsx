import React from 'react'
import renderer from 'react-test-renderer'
import BadgeBox from '@/components/badge-box'

describe('AuthorAvatar', () => {
  it('should render correct badge box', () => {
    const tags = ['typescript', 'devops']

    const component = renderer.create(<BadgeBox tags={tags} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
