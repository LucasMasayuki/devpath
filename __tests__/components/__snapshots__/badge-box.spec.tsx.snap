// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorAvatar should render correct badge box 1`] = `
<div
  className="css-r3u6lr"
>
  <span
    className="chakra-badge css-zib6pm"
  >
    typescript
  </span>
  <span
    className="chakra-badge css-14wwhjs"
  >
    devops
  </span>
</div>
`;
